Documentation/Developer/Index.rst

```diff
@@ -122,3 +122,55 @@ multiple ids separated by comma (e.g. 12,45) or an array (this is very useful if
 
    If you pass an array then both methods will return this array back without any modification. That
    makes it possible to call the methods in the getter without adding logic around it.
+
+
+Hooks / Events
+---------------
+
+Modify response object
+~~~~~~~~~~~~~~~~~~~~~~
+
+Use the `ModifyServiceBwResponseEvent` event to modify the response object of the Service BW API before it gets cached.
+The event dispatches before paginated requests are merged together.
+
+Add an event listener class that uses the event to modify the request.
+
+Example: Modify the URL of Service BW online forms (called: Prozesse):
+**********************************************************************
+
+.. code-block:: php
+
+   <?php
+
+   declare(strict_types=1);
+
+   namespace ThisIs\MySitePackage\Listener;
+
+   use JWeiland\ServiceBw2\Client\Event\ModifyServiceBwResponseEvent;
+
+   class ModifyServiceBwResponseListener
+   {
+       public function __invoke(ModifyServiceBwResponseEvent $event): void
+       {
+           if (strpos($event->getPath(), '/portal/leistungsdetails/') === 0) {
```

Comment:
I know it's just an example. But in TYPO3 we have StringUtility::beginsWith()

Reply:
Good idea 👍🏽